packages/admin/src/utils/fetchDataOrGetRedirect.ts

```diff
@@ -14,9 +14,8 @@ async function fetchDataOrGetRedirect<T>(
 
 const getRedirect = (error: unknown) => {
   if (error instanceof AxiosError) {
-    const unauthorized =
-      error?.response?.status === 401 || error?.response?.status === 403;
-    if (unauthorized) {
+    const forbidden = error?.response?.status === 403;
+    if (forbidden) {
       return {
```

Comment:
Could keep the logic for 401s and redirect to the login page in that scenario? Unless thats handled somehow elsewhere

Reply:
login page redirect is handled in the default case below these conditions